snapcraft/plugins/colcon.py

```diff
@@ -178,6 +178,14 @@ def schema(cls):
             "default": [],
         }
 
+        schema["properties"]["colcon-packages-ignore"] = {
```

Comment:
Please add docs for this new option as well, following the `colcon-packages` option on line 25 of this file.

Reply:
OK!